tools/depends/native/waylandpp-scanner-native/Makefile

```diff
@@ -0,0 +1,41 @@
+include ../../Makefile.include
+PREFIX=$(NATIVEPREFIX)
+PLATFORM=$(NATIVEPLATFORM)
+DEPS=../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=waylandpp
+VERSION=0.1.5
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+
+APP=$(PLATFORM)/wayland-scanner++
+CLEAN_FILES=$(ARCHIVE) $(PLATFORM)
+
+CMAKE_OPTIONS := -DCMAKE_TOOLCHAIN_FILE= -DBUILD_DOCUMENTATION=OFF -DBUILD_LIBRARIES=OFF -DBUILD_EXAMPLES=OFF -DBUILD_SCANNER=ON $(CMAKE_OPTIONS)
```

Comment:
Even if makefile variable evaluation might do the right thing here, please set
CMAKE_INSTALL_PREFIX=$(NATIVEPREFIX) to be explicit.

Reply:
I kind of dislike that we have to overwrite CMake arguments that are already specified (also applies to `CMAKE_TOOLCHAIN_FILE`). It will look weird in the log. Is there another way to solve this? Maybe have separate `Makefile.include` for native and target?

Reply:
set `CMAKE_FOR_BUILD` (or something similar) in `Makefile.include`